master/usage/troubleshooting/index.md

```diff
@@ -76,18 +123,7 @@ If you do not see this, please check the following.
 - There must not be iptables rules, or any kind of firewall, preventing
   communication between the hosts on TCP port 179.  (179 is the BGP port.)
 
-## Basic checks
-
-Running `ip route` shows what routes have been programmed. Routes from other hosts
-should show that they are programmed by bird.
-
-If your hosts reboot themselves with a message from `locksmithd` your cached CoreOS
-image is out of date.  Use `vagrant box update` to pull the new version.  I
-recommend doing a `vagrant destroy; vagrant up` to start from a clean slate afterwards.
-
-If you hit issues, please raise tickets. Diags can be collected with the
-`calicoctl node diags` command.  This should be run with superuser privileges,
-for example:
-
-        sudo calicoctl node diags
+## Further Assistance
 
+Still stuck? Collect diags with `sudo calicoctl node diags`, and raise an issu
```

Comment:
Worth giving some guidance on what information is useful.

e.g. orchestration system, platform, calico version, the actual issue.